src/Microsoft.Health.Fhir.Api/Configs/FhirServerConfiguration.cs

```diff
@@ -22,7 +23,7 @@ public class FhirServerConfiguration : IApiConfiguration
 
         public OperationsConfiguration Operations { get; } = new OperationsConfiguration();
 
-        public AuditConfiguration Audit { get; } = new AuditConfiguration("X-MS-AZUREFHIR-AUDIT-");
+        public AuditConfiguration Audit { get; } = new AuditConfiguration(SpecialValues.CustomAuditHeaderPrefix);
```

Comment:
>new AuditConfiguration(SpecialValues.CustomAuditHeaderPrefix) [](http://example.com/codeflow?start=51&length=61)

That's not right.
And it's not your fault, but if you touch this code we have chance to make it right.

It's either should be config or it should be const string, it can't be both. 
I would suggest to move to away from config to const